### ISSUE TITLE
Possibility to install without requirements?

### ISSUE THREAD

**Issue by claasd** [conan-io/conan#1276]
Hi,

I have a use case where I download a conan package an re-package it for a different package manager. 
for this, I don't need to install all the requirements of the package, just the package itself. Is there already a way to do this?

Something like
```
conan install MyFancyPackage/1.0.0@user/testing --skip_deps
```

Thanks!

**Comment by memsharded** [comment]
Yes, if you specify ``--all`` to conan install it will be a mere download of all binary packages for that recipe, but no dependency graph will be resolved. If you want just one of the binary packages, you can specify its ID too.
